src/components/Map/Map.js

```diff
@@ -0,0 +1,221 @@
+import React, {Component} from "react";
+import {connect} from "react-redux";
+import {withStyles} from "@material-ui/core/styles";
+import withWidth from "@material-ui/core/withWidth/index";
+import mapboxgl from "mapbox-gl";
+import "mapbox-gl/dist/mapbox-gl.css";
+import "./Map.css";
+import {
+  findMyLocation,
+  setBottomDrawerData,
+  toggleBottomDrawer,
+  setMap,
+  selectMapItem
+} from "../../redux/actions";
+import {
+  FIND_MY_LOCATION_ERROR,
+  FIND_MY_LOCATION_OUT_OF_BOUNDS,
+  FIND_MY_LOCATION_SELECT,
+  FIND_MY_LOCATION_SUCCESS
+} from "../../redux/constants";
+
+mapboxgl.accessToken =
+  "pk.eyJ1Ijoic3BhdGlhbGRldiIsImEiOiJjanh0bHczc2Qwdnd0M25udGQzZm9tcTBzIn0.MnBju8Y2wP6N6nFm4nNu7A";
+
+const styles = theme => ({
+  map: {
+    [theme.breakpoints.down("sm")]: {
+      position: "relative",
+      // toolbar height
+      top: "56px",
+      height: `calc(100% - 56px)`
+    },
+    [theme.breakpoints.up("md")]: {
+      position: "relative",
+      // 65 = toolbar height
+      top: "65px",
+      // 280 = left drawer width
+      left: "280px",
+      height: `calc(100% - 65px)`,
+      width: `calc(100% - 280px)`
+    },
+    [theme.breakpoints.up("lg")]: {
+      position: "relative"
+    }
+  }
+});
+
+const CHBP_CENTROID = [-122.31817942477232, 47.61410334875629];
```

Comment:
Likewise here, candidate for ./config.js file?

Reply:
💯